README.md

```diff
@@ -122,11 +122,35 @@ LOG_LEVEL=trace npm run test
 
 Alternatively, one may set `LOG_LEVEL` in the `.env` file.
 
-### Authentication
-
-The service validates a Bearer JSON Web Token (JWT) found in the `Authorization` HTTP header.
-
-See the `.env.example` for relevant variables.
+### Authentication and Authorization
+
+A valid Bearer JSON Web Token (Bearer JWT) is required in requests to the PDC service. The PDC officially reccomends using [KeyCloak](https://www.keycloak.org/) as the authentication provider, but supports any valid Bearer JWT provider. Please refer to [the relevant documentation for setup and configuration](https://www.keycloak.org/guides#getting-started)
+
+See `.env.example` for documentation on the necessary environment variables for Keycloak(or chosen authentication provider)
+
+#### An example with a Keycloak authentication and Swagger-UI
+
+We use Swagger to generate an interactive api interface as the default route for the service.
+From the Keycloak admin interface, e.g. https://my-host/admin:
+
+1. Add a realm e.g. "pdc" (avoid spaces in the name).
+2. Within the realm just added in step (1) add a client e.g. "pdc-service", with Client authentication off and Authorization off:
+   - Standard flow Checked,
```

Comment:
Ah, I see. This partially explains why it would work without adding a second client.

In our production Keycloak instance, we have this unchecked for `pdc-service` and checked for `pdc-openapi-docs`. On reflection, it has probably been a while, a long while, since anybody used the `pdc-service` client to get a session. So I have no objection to a single client for development purposes after all.